packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -1256,6 +1265,23 @@ class InspectorRowContent extends StatelessWidget {
                         // we wouldn't need this.
                         controller.requestFocus();
                       },
+                      onDoubleTap: isSummaryTree ? () async {
+                        controller.requestFocus();
+                        final List<int>? data =
+                            await controller.screenShot(row);
+                        unawaited(showDialog(
+                            context: context,
+                            builder: (BuildContext context) {
+                              if (data == null || data.isEmpty) {
+                                return Text("no data");
+                              }
+                              return  Container(
+                                width: 480,
```

Comment:
make these const and adjust based on the size of the window. If DevTools is displayed in a small embedded window, you probably want a smaller screenshot.

Reply:
ok i'll do it with the above